app/views/hq/orphans/index.html.erb

```diff
@@ -0,0 +1,41 @@
+<div class="panel-heading">
+  <div class="action_items">
+    <%= link_to('New Orphan', new_hq_orphan_path,
```

Comment:
It should not be possible to create a new orphan record this way, only via orphan list import. Please remove the link and ensure that `new_hq_orphan_path` is not routable .